docs/source/tutorials/submission.ipynb

```diff
@@ -0,0 +1,233 @@
+{
```

Comment:
<p>Text editing suggestions:</p><ul><li>Maybe explain what is a compute node, login node in terms of the HPC( we can either explain here or give a link)</li><li>the former on your workstation -&gt; the former on the user's workstation</li><li>How you deploy RADICAL-Pilot -&gt; How to deploy RADICAL-Pilot</li><li>(usually very little). -&gt; (usually minimal).</li><li> some user-denpendent configuration -&gt; some user-dependent configuration</li><li class="ql-indent-1">Also Maybe we can give a link to configuration files.</li><li>Also it would be good to give a link or small explanation about ssh. </li><li>two factor -&gt; two-factor  </li><li class="ql-indent-1">it could be good to give an example here.</li><li>passphrase protected -&gt; passphrase-protected  </li><li class="ql-indent-1">What is passphrase protected ssh ?</li><li> 2. warning maybe we can offer a solution such as tmux with a link</li><li>a ssh -&gt; an ssh</li><li>all the duration -&gt; the duration</li><li>This has the potential of leaving a orphan RP Agent running on the HPC platform, consuming allocation and failing to properly execute any new application task. -&gt; This could leave an orphan RP Agent running on the HPC platform, consuming allocation and failing to execute any new application task properly.</li><li>connection; and the -&gt; connection. The </li><li>if you  -&gt; if a user </li><li>You will have to set up an ssh and,  -&gt; The user will have to set up an ssh.</li><li> if you are not familiar with how to do that, check out this <a href="https://www.ssh.com/academy/ssh-keys#how-to-configure-key-based-authentication" target="_blank">guide</a>. -&gt; This <a href="https://www.ssh.com/academy/ssh-keys#how-to-configure-key-based-authentication" target="_blank">guide</a> shows how to set up an ssh key.</li><li>I would remove "Don't forget to configure your <code>ssh-agent</code> as, without it, RADICAL-Pilot will not work as it would have to enter your ssh key passphrase to access the HPC platform. " and add following as a note</li><li class="ql-indent-1"><strong>Note:</strong>&nbsp;&nbsp;RADICAL-Pilot will not work without configuring the ssh-agent, and it will require entering the user's ssh key passphrase to access the HPC platform.&nbsp;</li><li>You can  -&gt; The user can</li><li>within your application -&gt; within their application</li><li> Remote usernames, passwords, and keyfiles and can be set -&gt;  Remote usernames, passwords, and keyfiles can be set</li><li> For example, if you want to tell RADICAL-Pilot your user-id on the remote resource, you can use the following: -&gt;  For example, if a user wants to tell RADICAL-Pilot their user-id on the remote resource, the user can use the following</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/radical-cybertools/radical.pilot/pull/2874/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>